firebase-perf/src/main/java/com/google/firebase/perf/config/RemoteConfigManager.java

```diff
@@ -93,8 +94,12 @@ private RemoteConfigManager() {
         firebaseRemoteConfig == null
             ? new ConcurrentHashMap<>()
             : new ConcurrentHashMap<>(firebaseRemoteConfig.getAll());
-    this.appStartTimeInMs =
-        TimeUnit.MICROSECONDS.toMillis(FirebasePerfProvider.getAppStartTime().getMicros());
+    StartupTime startupTime = FirebaseApp.getInstance().get(StartupTime.class);
```

Comment:
can this be provided as a dependency instead of looking it up from firebaseApp?

Reply:
I've attempted to resolve this but it's more just shuffling the method around to be cleaner, `RemoteConfigManager` initializes itself during class init and I believe it'd be more of a refactor to change this behavior if passing from truly outside of the class is the goal.